src/ximea_ros_cam.cpp

```diff
@@ -1,5 +1,23 @@
 #include "ximea_ros_cam/ximea_ros_cam.hpp"
 
+namespace { // anon
+    // in leu of std::clamp
+    template<typename T> 
+    T clamp(const T& v, const T& lo, const T& hi)
+    {
+        return std::min(std::max(v, lo), hi);
+    }
+    int roundDown(int value, int increment)
+    {
+        return (value / increment) * increment;
```

Comment:
Are you ok with...  ```return int(value / increment) * increment;```  ...to keep it c++ style?
I take it this is to reassure a reader who might assume ```int / int``` is not an integral division.

Reply:
> Are you ok with... `return int(value / increment) * increment;` ...to keep it c++ style?
I am fine with it - the original line is fine as well if that suits you better.

> I take it this is to reassure a reader who might assume `int / int` is not an integral division.
The explicit `int` is meant to both show the reader and compiler no ambiguity in what the divide is meant to do. I am, of course being overly cautious here.

Speaking of being overly cautious, perhaps there should be a quick check here to avoid a divide by zero error =P

Reply:
There should really be that div 0 check, something like...
```return increment?  int(value / increment) * increment : value;``` 
or perhaps...
```return increment?  int(value / increment) * increment : 0;```